test/index.ts

```diff
@@ -78,7 +82,8 @@ describe('retry-axios', () => {
     assert.fail('Expected to throw');
   });
 
-  it('should retry at least the configured number of times', async () => {
+  it('should retry at least the configured number of times', async function () {
+    this.timeout(10000);
```

Comment:
Had to change the timeouts to the old function style after getting a weird error on the ci tests - https://github.com/JustinBeckwith/retry-axios/pull/142/checks?sha=f6f6c8d1bbe8b2a6da5301ee457fd6fe0970e121